Standalone Demo/Standalone Demo/Examples/Interstitial.swift

```diff
@@ -19,21 +19,25 @@ class Interstitial: UIViewController {
     @IBAction func loadAdTouchUpInside(_ sender: UIButton) {
         activityIndicator.startAnimating()
         showAdButton.isHidden = true
-// Step 4: Set an optional skipOffset value
-        interstitialAd.setSkipOffset(5)
-// Step 5: Request a HyBidAd
+// Step 4: Set HTML Interstitial skipOffet (Optional)
+        HyBid.setHTMLInterstitialSkipOffset(2)
```

Comment:
Wrong skipOffset methods are called here. We should use the methods that are available in  `HyBidInterstitialAd` class. _(Both for HTML and Video)_